brushfire-core/src/main/scala/com/stripe/brushfire/Injections.scala

```diff
@@ -124,57 +124,49 @@ object JsonInjections {
             obj.put("distribution", toJsonNode(target))
             obj
 
-          case SplitNode(children) =>
-            val ary = JsonNodeFactory.instance.arrayNode
-            children.foreach {
-              case (feature, predicate, child) =>
-                val obj = JsonNodeFactory.instance.objectNode
-                obj.put("feature", toJsonNode(feature))
-                obj.put("predicate", toJsonNode(predicate))
-                obj.put("display", toJsonNode(Predicate.display(predicate)))
-                obj.put("children", toJsonNode(child)(nodeJsonNodeInjection))
-                ary.add(obj)
-            }
-            ary
+          case SplitNode(p, k, lc, rc, _) =>
+            val obj = JsonNodeFactory.instance.objectNode
+            obj.put("predicate", toJsonNode(p)(predicateJsonNodeInjection))
+            obj.put("key", toJsonNode(k))
+            obj.put("left", toJsonNode(lc)(nodeJsonNodeInjection))
+            obj.put("right", toJsonNode(rc)(nodeJsonNodeInjection))
+            obj
         }
 
-        def tryChild(node: JsonNode, property: String) = Try {
+        def tryChild(node: JsonNode, property: String): Try[JsonNode] =
+          Try {
+            val child = node.get(property)
+            assert(child != null, property + " != null")
```

Comment:
I know this existed before, but could we make this `require` instead? It is reasonable to expect that we'll be given bad JSON, where as `assert`s are more meant for sanity checks that may possibly be elided.